.github/workflows/osrm-backend-docker.yml

```diff
@@ -0,0 +1,86 @@
+name: build and publish container image
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch:
```

Comment:
Was this just for testing?

Reply:
Looks good otherwise 👍

Reply:
never got it working since it aparently only activates when on main branch
so ill let you decide whether to keep it